### ISSUE TITLE
requestBody description ignored

### ISSUE THREAD

**Issue by jayvdb** [juhaku/utoipa#700]
We have a few actix routes which has a requestBody description that is being ignored in v3.4.2. 
This is a minor problem.
i.e. this used to cause an openapi 'description' on the `requestBody`, but now it has been removed.

```rust
    request_body(
        content = [u8],
        content_type = "application/octet-stream",
       description = "Binary data to be transferred",
    ),
```

We are also seeing a lot of blanks, i.e. `description: ""`, always on `requestBody`, so seems related.

**Comment by juhaku** [comment]
Yes, this will be fixed in #701

**Comment by juhaku** [comment]
Of course I forgot to test how it behaves with already defined request bodies. :facepalm: There is a support for automatically resolved request bodies but from #701 onwards it will only be resolved if request body hasn't been defined manually. This is a bit rough approach and better would be to allow granular merge between automatic and manually defined one. Yet this is something to be addressed in future releases.
